source/Calamari.Common/Calamari.Common.csproj

```diff
@@ -1,16 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
-        <TargetFrameworks>net452;netstandard2.1;net40</TargetFrameworks>
+        <TargetFrameworks>net452;netstandard2.1</TargetFrameworks>
```

Comment:
Should this be bumped to `net462`?

Reply:
nice good find. Not sure how i didnt pick that up in the find all usages :)